src/main/java/juniorJavaDevClass/Labs/StudentDatabase/Student.java

```diff
@@ -0,0 +1,105 @@
+package juniorJavaDevClass.Labs.StudentDatabase;
+
+/*
+Lab 2 - Object-Oriented Programming
+Create a student Database with the following attributes:
+    - New Student constructor that takes name and SSN in the arguments
+    - Automatically create an email ID based on the name
+    - Set a private static ID
+    - Generate a user ID that is combination fo Static ID, random 4-digit number between 1000 and 9000, and the last 4 of SSN
+    - Methods: enroll(), pay(), toString(), showCourses()
+    use encapsulation to set variables (phone, city, state)
+ */
+
+import java.util.Locale;
+
+public class Student {
+
+    private static int iD = 0;
+    private String name;
+    private String SSN;
+    private String email;
+    private String userID;
+    private String courses = "";
+    private int courseCost = 1000;
+    private int balance = 0;
+    private String phone;
+    private String city;
+
+    public String getPhone() {
+        return phone;
+    }
+
+    public void setPhone(String phone) {
+        this.phone = phone;
+    }
+
+    public String getCity() {
+        return city;
+    }
+
+    public void setCity(String city) {
+        this.city = city;
+    }
+
+    public String getState() {
+        return state;
+    }
+
+    public void setState(String state) {
+        this.state = state;
+    }
+
+    private String state;
+
+    // constructor
+    public Student (String name, String SSN) {
```

Comment:
this seems to have been part of the book, but just an FYI, constructors are usually placed below the members of the class, so this would usually go on line 29.